basics/shutdown-server/src/main.rs

```diff
@@ -29,7 +29,7 @@ async fn main() -> std::io::Result<()> {
     let server = HttpServer::new(move || {
         // give the server a Sender in .data
         App::new()
-            .data(tx.clone())
+            .app_data(tx.clone())
```

Comment:
the equivalent to deprecated `.data(val)` is `.app_data(Data::new(val))`

Reply:
Will fix this.